res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -232,14 +232,24 @@ limitations under the License.
 
         display: flex;
         align-items: center;
-        justify-content: center;
+        justify-content: start;
         padding: 5px 0;
 
         .mx_EventTile_avatar {
             position: static;
             order: -1;
             margin-right: 5px;
         }
+
+        .mx_EventTile_e2eIcon {
+            margin-left: 9px;
+        }
+
+        .mx_EventTile_line > a {
+            right: auto;
+            top: -15px;
+            left: -68px;
```

Comment:
Why do we need this specific positioning here?

Reply:
Event that aren't within a bubble do not have the same height nor padding as other event.
Due to the specificity of the CSS elsewhere it forces me to override it here